source/def-stdlib.js

```diff
@@ -678,5 +678,65 @@ if (doc) doc.children.push({
 	`,
 	"children": [
 	]},
+
+	{
+		"id": "audio",
+		"name": "Audio playback",
+		"title": "Audio playback",
+		"content": `
+			<p>TScript supports playback of audio. The classes <code class="code">MonoAudio</code> and <code class="code">StereoAudio</code> allow the playback of arbitrary buffers consisting of real with values from -1 to 1.</p>
+			<h3>Constructors</h3>
+			<table class="methods">
+			<tr><th>MonoAudio</th><td>
+				<code class="code">MonoAudio(buffer, sampleRate)</code> where <code class="code">buffer</code> is an array containing the samples to be played, <code class="code">sampleRate</code> specifies the rate they are to be played at in Hz.
+			</td></tr>
+			<tr><th>StereoAudio</th><td>
+				<code class="code">StereoAudio(leftBuffer, rightBuffer, sampleRate)</code> provides the same functionality as above, but allows playback of stereo audio.
+			</td></tr>
+			</table>
+			<h3>Functions</h3>
+			<tscript>
+			function play(){}
+			function pause(){}
+			function setPlaybackRate(speed){}
+			</tscript>
+			Are available on <code class="code">MonoAudio</code> and <code class="code">StereoAudio</code>, they behave as expected.
+			<h3> Example</h3>
+			<p> The following code plays two different tones on the left and right channel. <code class="code">l_freq</code> and <code class="code"> r_freq</code> 
+			and the <code class="code"> sampleRate</code> can be modified to alter the tones.
+			<tscript>
+				use namespace math;
+				use namespace audio;
+
+				var lenInSeconds = 2;
+				var l_freq = 180;
+				var r_freq = 1000;
+				var sampleRate = 48000;
+
+				var l_sf = (l_freq / sampleRate) * pi(); 
+				var r_sf = (r_freq / sampleRate) * pi(); 
+
+				var l_samples = [];
+				var r_samples = [];
+
+				var c = 0;
+				for var i in 0:(sampleRate*lenInSeconds) do{
+					l_samples.push(sin(c));
```

Comment:
I find sin(l_sf * i) more natural, and I would even go for sin(2 * math.pi() * l_sf * i), since that would turn l_sf into a "proper" frequency (measured in Hz).

Reply:
`l_freq` was supposed to be the frequency in Hz and  `l_sf` is the 'sample frequency' (i.e. the frequency per sample, not to be confused with sample rate). But i can see how this would be confusing as an example.